### PR TITLE
tikv: Fix some words for TiKV API v2

### PR DESCRIPTION
**Pull request by pingyu**
Signed-off-by: pingyu <yuping@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- English translation of https://github.com/pingcap/docs-cn/pull/11959

See [rendered](https://github.com/pingyu/docs/blob/tikv-api-v2-fix-en/tikv-configuration-file.md#api-version-new-in-v610).

Please also note that some links to tikv.org is under developing, and will be finished in several days.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by haojinming — APPROVED** [review]
LGTM

**Review by zeminzhou — APPROVED** [review]
LGTM～

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -414,20 +414,21 @@ Configuration items related to storage.
...
-        + Since v6.2.0, Change Data Capture (CDC) for RawKV is supported using the component [TiKV-CDC](https://github.com/tikv/migration/tree/main/cdc).
+        + Since v6.2.0, Change Data Capture (CDC) for RawKV is supported. Please refer to [RawKV CDC](https://tikv.org/docs/dev/concepts/explore-tikv-features/cdc/).
 + Default value: `1`
 
 > **Warning:**
 
-> - You can set the value of `api-version` to `2` **only when** deploying a new TiKV cluster. **Do not** modify the value of this configuration item in an existing TiKV cluster. TiKV clusters with different `api-version` values use different data formats. Therefore, if you modify the value of this item in an existing TiKV cluster, the cluster will store data in different formats and causes data corruption. It will raise the "unable to switch storage.api_version" error when you start the TiKV cluster.
+> - As the difference of storage format between API V1 and API V2, you can enable or disable API V2 smoothly **ONLY** when no other but TiDB data existing in TiKV. In other scenarios, you need to deploy a new cluster, and migrate data by [RawKV BR](https://tikv.org/docs/dev/concepts/explore-tikv-features/backup-restore/).
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:431]:
```suggestion
> - As the different underlying storage format between API V1 and API V2, you can enable or disable API V2 smoothly **only** when there are only TiDB data existing in TiKV. In other scenarios, you need to deploy a new cluster, and migrate data using [RawKV Backup & Restore](https://tikv.org/docs/dev/concepts/explore-tikv-features/backup-restore/).
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -414,20 +414,21 @@ Configuration items related to storage.
 
 ### `api-version` <span class="version-mark">New in v6.1.0</span>
 
-+ The storage format and interface version used by TiKV when TiKV serves as the raw key-value store.
++ The storage format and interface version used by TiKV when TiKV serves as the RawKV store.
 + Value options:
     + `1`: Uses API V1, does not encode the data passed from the client, and stores data as it is. In versions earlier than v6.1.0, TiKV uses API V1 by default.
     + `2`: Uses API V2:
         + The data is stored in the MVCC (Multi-Version Concurrency Control) format, where the timestamp is obtained from PD (which is TSO) by tikv-server.
+        + Data is scoped according to different usage and supports co-existence of TiDB, Transactional KV, and RawKV applications in a single cluster.
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply tikv-configuration-file.md:422]: LGTM

**Pull request merged**